src/main/java/com/mumfrey/worldeditcui/config/CUIConfiguration.java

```diff
@@ -29,29 +33,29 @@
 
-	@Expose private boolean debugMode = false;
-	@Expose private boolean ignoreUpdates = false;
-	@Expose private boolean promiscuous = false;
-	@Expose private boolean alwaysOnTop = false;
-	@Expose private boolean clearAllOnKey = false;
-	
-	@Expose private Colour cuboidGridColor        = ConfiguredColour.CUBOIDBOX.getDefault();
-	@Expose private Colour cuboidEdgeColor        = ConfiguredColour.CUBOIDGRID.getDefault();
-	@Expose private Colour cuboidFirstPointColor  = ConfiguredColour.CUBOIDPOINT1.getDefault();
-	@Expose private Colour cuboidSecondPointColor = ConfiguredColour.CUBOIDPOINT2.getDefault();
-	@Expose private Colour polyGridColor          = ConfiguredColour.POLYGRID.getDefault();
-	@Expose private Colour polyEdgeColor          = ConfiguredColour.POLYBOX.getDefault();
-	@Expose private Colour polyPointColor         = ConfiguredColour.POLYPOINT.getDefault();
-	@Expose private Colour ellipsoidGridColor     = ConfiguredColour.ELLIPSOIDGRID.getDefault();
-	@Expose private Colour ellipsoidPointColor    = ConfiguredColour.ELLIPSOIDCENTRE.getDefault();
-	@Expose private Colour cylinderGridColor      = ConfiguredColour.CYLINDERGRID.getDefault();
-	@Expose private Colour cylinderEdgeColor      = ConfiguredColour.CYLINDERBOX.getDefault();
-	@Expose private Colour cylinderPointColor     = ConfiguredColour.CYLINDERCENTRE.getDefault();
-	@Expose private Colour chunkBoundaryColour    = ConfiguredColour.CHUNKBOUNDARY.getDefault();
-	@Expose private Colour chunkGridColour        = ConfiguredColour.CHUNKGRID.getDefault();
-
-	private static Map<String, Object> configArray = new HashMap<String, Object>();
-	
+	private boolean debugMode = false;
+	private boolean ignoreUpdates = false;
+	private boolean promiscuous = false;
+	private boolean alwaysOnTop = false;
+	private boolean clearAllOnKey = false;
+
+	private Colour cuboidGridColor		= ConfiguredColour.CUBOIDBOX.getDefault();
+	private Colour cuboidEdgeColor		= ConfiguredColour.CUBOIDGRID.getDefault();
+	private Colour cuboidFirstPointColor  = ConfiguredColour.CUBOIDPOINT1.getDefault();
+	private Colour cuboidSecondPointColor = ConfiguredColour.CUBOIDPOINT2.getDefault();
+	private Colour polyGridColor		  = ConfiguredColour.POLYGRID.getDefault();
+	private Colour polyEdgeColor		  = ConfiguredColour.POLYBOX.getDefault();
+	private Colour polyPointColor		 = ConfiguredColour.POLYPOINT.getDefault();
+	private Colour ellipsoidGridColor	 = ConfiguredColour.ELLIPSOIDGRID.getDefault();
+	private Colour ellipsoidPointColor	= ConfiguredColour.ELLIPSOIDCENTRE.getDefault();
+	private Colour cylinderGridColor	  = ConfiguredColour.CYLINDERGRID.getDefault();
+	private Colour cylinderEdgeColor	  = ConfiguredColour.CYLINDERBOX.getDefault();
+	private Colour cylinderPointColor	 = ConfiguredColour.CYLINDERCENTRE.getDefault();
+	private Colour chunkBoundaryColour	= ConfiguredColour.CHUNKBOUNDARY.getDefault();
+	private Colour chunkGridColour		= ConfiguredColour.CHUNKGRID.getDefault();
```

Comment:
I think either those equal signs should be aligned properly, or not being aligned at all (just 1 space before/after equals sign - as usually)